play/PlayerRules/CharacterCreation/04_01_Weapons.txt

```diff
@@ -398,11 +398,101 @@ unless otherwise stated.
 
 ===== Level 1 Weapons =====
 
----- 
+---- Matthews Model 2x12	$1029
+Damage, Miss chance:
+	140 damage 3 to miss out to 10m
+	105 damage 4 to miss out to 20m
+	Slug only, 115 damage 5 to miss out to 50m
+Automatic Accuracy penalty: 2 - 1 for every 9 Strength
+Mag Size:	2 shells of 12 gauge. $10 per pair.
+Reload DC:		38%
+Fire Modes:	Semi, Auto
+Automatic Fire Rate: 		2
+Armor Pierce Level:		1
+Reflex Modifier:			-8
+Suppression Level:		Light
+Max Speed Penalty: 		1m/action

```

Comment:
speed penalties apply directly to your 'movement speed' stat. In other words a -1 movement speed penalty is 1m/ two actions